core/primitives/src/account.rs

```diff
@@ -55,14 +67,25 @@ impl Account {
         locked: Balance,
         code_hash: CryptoHash,
         storage_usage: StorageUsage,
+        protocol_version: ProtocolVersion,
     ) -> Self {
-        Account {
+        #[cfg(not(feature = "protocol_feature_add_account_versions"))]
+        let _ = protocol_version;
```

Comment:
```suggestion
```

I think the `checked_feature` macro does this internally

Reply:
becuase without this feature we have no corresponding field it would not